server/pkg/create-user-db

```diff
@@ -30,7 +31,7 @@ This is meant to be used by Soledad Server.
 parser = argparse.ArgumentParser(description=description)
 parser.add_argument('dbname', metavar='user-d34db33f', type=str,
                     help='database name on the format user-{uuid4}')
-NETRC_PATH = '/etc/couchdb/couchdb-admin.netrc'
+NETRC_PATH = load_configuration('/etc/leap/soledad-server.conf')
```

Comment:
I don't understand this change. `NETRC_PATH` is used later on in a `os.path.exists`, but `load_configuration` returns a dict. Are you sure that this code works?